### PR TITLE
Update NPU example readme

### PR DESCRIPTION
**Pull request by plusbang**
## Description

Update NPU readme

### CODE REVIEW

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 1]
```diff
@@ -77,45 +76,9 @@ Inference time: xxxx s
...
 ```
 
-## Example 2: Predict Tokens using `generate()` API using multi processes
+## 4. Run Optimized Models (Experimental)
 In the example [llama2.py](./llama2.py) and [qwen2.py](./qwen2.py), we show an experimental support for a Llama2 / Qwen2 model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimization and fused decoderlayer optimization on Intel NPUs.
 
-> [!IMPORTANT]
-> To run Qwen2 and Llama2 with IPEX-LLM on Intel NPUs, we recommend using version **32.0.100.2540** for the Intel NPU.
-> 
-> Go to https://www.intel.com/content/www/us/en/download/794734/825735/intel-npu-driver-windows.html to download and unzip the driver. Then follow the same steps on [Requirements](#0-requirements).
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:86]: Do we still need this notice?

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md [hunk 2]
```diff
@@ -77,45 +76,9 @@ Inference time: xxxx s
 done
 ```
 
-## Example 2: Predict Tokens using `generate()` API using multi processes
+## 4. Run Optimized Models (Experimental)
 In the example [llama2.py](./llama2.py) and [qwen2.py](./qwen2.py), we show an experimental support for a Llama2 / Qwen2 model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimization and fused decoderlayer optimization on Intel NPUs.
```
Comment (jason-dai) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:80]: The examples below shows how to run the optimized model implementations on Intel NPU, including
- [Llama2-7B](./llama2.py)
- [Qwen2-1.5B](./qwen2.py)

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md (hunk 1 above)
Comment (plusbang) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md:86]: > Do we still need this notice?

Our troubleshooting (https://github.com/intel-analytics/ipex-llm/blob/4fcffc40503bd92aad164bb2d4a9cbb69c66342d/python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md#troubleshooting) involves transpose value setting workaround. And 32.0.100.2540 is mainly verified on MTL.

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md (hunk 2 above)
Comment (plusbang) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md]: > The example below shows how to run the **_optimized model implementations_** on Intel NPU, including
> 
> * [Llama2-7B](./llama2.py)
> * [Qwen2-1.5B](./qwen2.py)

Have updated.

**Review by jason-dai — APPROVED** [review]
LGTM

**Pull request merged**